.travis.yml

```diff
@@ -1,21 +1,6 @@
 language: python
 python:
-  - 2.7
-env:
-#  - PLONE_VERSION=4.1
-#  - PLONE_VERSION=4.2
-  - PLONE_VERSION=4.3
-#  - PLONE_VERSION=5.0
-#matrix:
-#  include:
-#    - python: 2.6
-#      env: PLONE_VERSION=4.0
-#    - python: 2.6
-#      env: PLONE_VERSION=4.1
-#  allow_failures:
-#    - python: 2.6
-#      env: PLONE_VERSION=4.0
+  - 3.6
```

Comment:
Is travis still a thing?

Reply:
I don't use travis. Normally I use gitlab. But after the change I saw how github works with github actions. Maybe, I should delete this file?